osi_streamingupdate.proto

```diff
@@ -0,0 +1,74 @@
+syntax = "proto2";
+
+option optimize_for = SPEED;
+
+import "osi_version.proto";
+import "osi_common.proto";
+import "osi_object.proto";
+import "osi_trafficlight.proto";
+import "osi_hostvehicledata.proto";
+
+package osi3;
+
+//
+// \brief The streaming update interface enables simulation entities to send
+// partial updates to other modules that favor performance (especially latency)
+// over data completeness/consistency (e.g. visualization applications).
+//
+// Static and/or non-relevant objects can be omitted as required for the
+// specific use case. Adding an object's unique id to the repeated field \c
+// obsolete_id indicates that it will no longer be updated from then on.
+//
+// \note The receiver of partial streaming update messages can only rely on the
+// most up-to-date information at the corresponding timestamp.
+//
+message StreamingUpdate
+{
+    // The interface version used by the sender.
+    //
+    optional InterfaceVersion version = 1;
+
+    // The data timestamp where the information of contained objects is calculated. 
+    // 
+    // Zero time is arbitrary but must be identical for all messages. 
+    // Zero time does not need to coincide with the UNIX epoch. 
+    // Recommended is the starting time point of the simulation.
+    //
+    optional Timestamp timestamp = 2;
+
+    // The list of stationary objects (excluding traffic signs and traffic
+    // lights).
+    //
+    repeated StationaryObject stationary_object_update = 4;
+
+    // The list of moving objects. 
+    //
+    repeated MovingObject moving_object_update = 5;
+
+    // The list of traffic signs.
+    //
+    repeated TrafficSign traffic_sign_update = 6;
+
+    // The list of traffic lights.
+    //
+    repeated TrafficLight traffic_light_update = 7;
+
+    // Conditions of the environment.
+    //
+    optional EnvironmentalConditions environmental_conditions_update = 12;
+
+    // Host vehicle data.
+    //
+    // Host vehicle data is data that the host vehicle knows about itself,
+    // e.g. from location sensors, internal sensors and ECU bus data, etc.,
+    // that is made available to sensors as input.
+    //
+    repeated HostVehicleData host_vehicle_data_update = 29;
```

Comment:
Isn't HostVehicleData already a top level interface? The OSI documentation does not list HostVehicleData under the section [Top-level interfaces](https://opensimulationinterface.github.io/osi-documentation/#_top_level_interfaces) but it already contains the version/timestamp information. So it seems to me that it was maybe meant to be. Couldn't this then be separated out so that this streaming update message tends to have a more global view on things?